W-18138142-SalesforceConnectorUpdate-SG (#585)
mulesoft/docs-composer: en/mulesoft/modules/ROOT/pages/ms_composer_salesforce_reference.adoc

Create new record::

Create a standard or custom record.
+

> While creating a new record, any rules associated with the record are triggered as per the user's role.

Reviewer: ```suggestion
While creating a new record, any rules associated with the record are triggered based on the user's role.
```
